test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml

```diff
@@ -33,6 +33,7 @@
         <menuitem icon=~/Person />
     </menu>
     <object data="~/Person" archive="~/Person/@Model.Name" data="~/Person" archive="~/Person/@Model.Name" />
+    <object archive="@if(Model.Name != null) { <text>~/Person/@Model.Name</text> } else { <text>~/Person</text> } " />
```

Comment:
Will add. Only reason why it doesn't complain about the lacking `@await` is due to the supression above `ExecuteAsync`.